src/test/kotlin/no/nav/syfo/client/pdfgen/PdfGenClientSpek.kt

```diff
@@ -13,20 +13,31 @@ class PdfGenClientSpek : Spek({
     val pdfGenClient = externalMockEnvironment.pdfgenClient
 
     describe("${PdlClient::class.java.simpleName}: navn") {
-        it("returns bytearray of pdf when requestBody is set correct") {
-            runBlocking {
-                val varselPdfDTO = VarselPdfDTO.create(
-                    mottakerNavn = UserConstants.PERSON_FULLNAME,
-                    mottakerPersonIdent = UserConstants.ARBEIDSTAKER_PERSONIDENT,
-                    documentComponents = emptyList(),
+        it("returns bytearray of pdf for forhandsvarsel") {
+            val forhandsvarselPdfDTO = ForhandsvarselPdfDTO.create(
+                mottakerNavn = UserConstants.PERSON_FULLNAME,
+                mottakerPersonIdent = UserConstants.ARBEIDSTAKER_PERSONIDENT,
+                documentComponents = emptyList(),
+            )
+
+            val pdf = runBlocking {
+                pdfGenClient.createForhandsvarselPdf(
+                    callId = "",
+                    forhandsvarselPdfDTO = forhandsvarselPdfDTO,
                 )
+            }
 
-                val pdf = pdfGenClient.createForhandsvarselPdf(
+            pdf shouldBeEqualTo UserConstants.PDF_FORHANDSVARSEL
+        }
+        it("returns bytearray of pdf for vurdering") {
+            val pdf = runBlocking {
+                pdfGenClient.createVurderingPdf(
                     callId = "",
-                    varselPdfDTO = varselPdfDTO,
+                    VurderingPdfDTO.create(emptyList()),
```

Comment:
Hvorfor empyList her? 🤔

Reply:
Er vel mest latskap siden det ikke har noe å si for testen... eller tenker du at vi burde validere at ikke tom liste i pdfgenclient?

Reply:
Legger på validering av lista i apiet i videre PR, men kan eventuelt vurdere å legge det til i pdfgenclient også da.